src/blacken_docs/__init__.py

```diff
@@ -338,14 +338,16 @@ def main(argv: Sequence[str] | None = None) -> int:
         "--rst-literal-blocks",
         action="store_true",
     )
+    parser.add_argument("--pyi", action="store_true")
     parser.add_argument("filenames", nargs="*")
     args = parser.parse_args(argv)
 
-    black_mode = black.FileMode(
+    black_mode = black.Mode(
```

Comment:
Just a drive-by change, since I spotted that this is just an alias for backwards compatibility in Black these days: https://github.com/psf/black/blob/c98fc0c128e1a79a776b62046acaa28685a31089/src/black/__init__.py#L113-L114

Reply:
Sure, thanks!